screens/MainScreen.tsx

```diff
@@ -1,346 +1,404 @@
+  //   left: '94.54%',
+  //   opacity: 0.4,
+  //   right: '0%',
+  // },
+  // capacity: {
+  //   height: '64.71%',
+  //   width: '73.99%',
+  //   top: '17.65%',
+  //   right: '17.79%',
+  //   bottom: '17.65%',
+  //   left: '8.22%',
+  //   borderRadius: 1,
+  // },
+  // battery: {
+  //   height: '25.76%',
+  //   width: '6.49%',
+  //   top: '39.39%',
+  //   right: '3.82%',
+  //   bottom: '34.85%',
+  //   left: '89.69%',
+  //   position: 'absolute',
+  // },
+  // wifiIcon: {
+  //   width: 15,
+  //   height: 11,
+  // },
+  // lte: {
+  //   top: 17,
+  //   left: 312,
+  //   fontSize: 11,
+  //   letterSpacing: -1,
+  //   fontWeight: '500',
+  //   fontFamily: FontFamily.sFProTextMedium,
+  //   width: 25,
+  //   height: 12,
+  //   display: 'none',
+  // },
+  // cellularIcon: {
+  //   height: '24.24%',
+  //   width: '4.53%',
+  //   top: '40.15%',
+  //   right: '17.07%',
+  //   bottom: '35.61%',
+  //   left: '78.4%',
+  // },
+  // time: {
+  //   marginTop: -3.5,
+  //   top: '50%',
+  //   letterSpacing: 0,
+  //   fontWeight: '600',
+  //   fontFamily: FontFamily.sFProTextSemibold,
+  //   fontSize: FontSize.size_mini,
+  //   left: '0%',
+  //   width: '100%',
+  // },
+  // barsStatusBarIphoneT: {
+  //   height: '47.73%',
+  //   width: '14.4%',
+  //   top: '15.91%',
+  //   right: '80%',
+  //   bottom: '36.36%',
+  //   left: '5.6%',
+  //   position: 'absolute',
+  // },
+  // barsStatusBarIphoneL: {
+  //   height: '5.42%',
+  //   bottom: '94.58%',
+  //   left: '0%',
+  //   right: '0%',
+  //   top: '0%',
+  //   position: 'absolute',
+  //   width: '100%',
+  // },
   voiceFabIcon: {
-    top: 698,
-    left: 144,
+    // top: 698,
+    // left: 144,
     width: 88,
     height: 88,
-    position: "absolute",
+    position: 'absolute',
+    bottom: 0,
+    right: 144,
   },
   helloHopeYouContainer: {
-    height: "75.39%",
-    width: "85.53%",
-    left: "14.35%",
+    height: '75.39%',
+    width: '85.53%',
+    left: '14.35%',
     fontSize: 18,
     fontFamily: FontFamily.nunitoSansRegular,
-    top: "0%",
+    top: '0%',
   },
   systemUiconswrite: {
     top: 11,
     borderRadius: 10,
     width: 36,
     height: 48,
-    overflow: "hidden",
+    overflow: 'hidden',
   },
   pm: {
     right: 0,
     bottom: 0,
-    fontWeight: "300",
+    fontWeight: '300',
     fontFamily: FontFamily.nunitoSansLight,
     width: 85,
     height: 16,
     fontSize: FontSize.size_mini,
   },
   translatedtextitem: {
     left: 0,
-    position: "absolute",
+    position: 'absolute',
     top: 0,
   },
   frameWrapper: {
     padding: 10,
     maxWidth: 375,
     top: 0,
-    overflow: "hidden",
+    overflow: 'hidden',
     flex: 1,
   },
   mainScreenInner: {
     top: 77,
     height: 584,
   },
-  mainScreenIcon: {
-    overflow: "hidden",
-    height: 812,
-    width: "100%",
-  },
+  // mainScreenIcon: {
+  //   overflow: 'hidden',
+  //   height: 812,
+  //   width: '100%',
+  // },
```

Comment:
Please Remove the commented code, remember not to push any unused code to the PR